From a264ca4e91f59d49aa1ccbd6cee2182960b6074f Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Mon, 3 Jun 2019 06:44:13 -0700
Subject: [PATCH 09/11] sepol: Allow Bluetooth hal to access
 vendor.qcom.bluetooth.soc

I broke this with the previous commit

Change-Id: I0dc7635a5cbe0e9e880d7889df9b71a135d02239
---
 qcom/common/hal_bluetooth.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 qcom/common/hal_bluetooth.te

diff --git a/qcom/common/hal_bluetooth.te b/qcom/common/hal_bluetooth.te
new file mode 100644
index 0000000..238b9f6
--- /dev/null
+++ b/qcom/common/hal_bluetooth.te
@@ -0,0 +1 @@
+get_prop(hal_bluetooth, exported3_default_prop)
-- 
2.17.1

